src/storage/src/hummock/sstable/block.rs

```diff
@@ -37,11 +37,14 @@ pub struct Block {
     data_len: usize,
     /// Restart points.
     restart_points: Vec<u32>,
+
+    table_id: u32,
```

Comment:
u32 -> `TableId` ?

shall we need to change u32 -> u64 cc @Little-Wallace

Reply:
This comment is not resolved